Modules/Devices/src/Devices.Infrastructure/PushNotifications/DirectPush/FirebaseCloudMessaging/NotificationContent.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+using Enmeshed.DevelopmentKit.Identity.ValueObjects;
+using Enmeshed.Tooling;
+using Newtonsoft.Json;
+
+namespace Backbone.Modules.Devices.Infrastructure.PushNotifications.DirectPush.FirebaseCloudMessaging;
+
+public class NotificationContent
```

Comment:
I just thought about it, and we should actually use the same class for AzureNotificationHub and the direct push. Sorry for the inconveniences. So you can move one of the two classes to the push notifications root folder and delete the other one.